code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -651,18 +661,6 @@ func (sc *StorageSmartContract) commitBlobberConnection(
 	alloc.Stats.UsedSize += commitConnection.WriteMarker.Size
 	alloc.Stats.NumWrites++
 
-	// UpdateItem saved_data on storage node
```

Comment:
@Sriep this is the only change we have in this PR.. rest are refactoring

Reply:
this was a non-required step which we have deleted

Reply:
So we will get the total saved data from event db? If the `SavedData` will be used by smart contracts for other calculation, then perhaps we need to keep it updated, otherwise, we can remove it.